src/_posts/2017-03-16-python-sdk-release.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Python SDK released in Apache Beam 0.6.0"
+date:   2017-03-16 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - altay
+---
+
+Apache Beam’s latest release, version [0.6.0](https://beam.apache.org/get-started/downloads/), introduces a new SDK -- this time, for the Python programming language. The Python SDK joins the Java SDK as the second implementation of the Beam programming model.
+
+<!--more-->
+
+The Python SDK incorporates all of the main concepts of the Beam model, including ParDo, GroupByKey, Windowing, and others. It features extensible IO APIs for writing bounded sources and sinks, and provides built-in implementation for reading and writing Text, Avro, and TensorFlow record files, as well as connectors to Google BigQuery and Google Cloud Datastore.
+
+There are two runners capable of executing pipelines written with the Python SDK today: [Direct Runner](https://beam.apache.org/documentation/runners/direct/) and [Dataflow Runner](https://beam.apache.org/documentation/runners/dataflow/). These runners are currently limited to batch only execution. Upcoming features will shortly bring the benefits of the Python SDK to additional runners.
```

Comment:
... and Dataflow Runner, both of which are currently limited to batch execution only.